pipelines/ci.yml

```diff
@@ -17,3 +18,9 @@ steps:
   inputs:
     targetType: filePath
     filePath: ./scripts/build.sh
```

Comment:
let's discuss offline about this build script, there's some duplicate work here

Reply:
@mydiemho sounds good. not super high pri right now anyway